src/composables/events/FileEvent.js

```diff
@@ -52,6 +52,12 @@ const ExpandFolderEvent = new Subject();
  */
 const UpdateFileEvent = new Subject();
 
+/**
+ * Represent a rxjs Event object to emit and to receive events file global saving.
```

Comment:
`...to receive events about files global saving.`

Reply:
done